docker-compose-test.yml

```diff
@@ -24,7 +25,7 @@ services:
     command: server --console-address :9001 /data
     networks:
       - hmpps_int
-    container_name: aws-s3-mock
+    container_name: s3
```

Comment:
I'm almost tempted to suggest maybe call this `minio` as we use that hostname in few places from within `docker-compose` but I'm not too fussy about it.

Reply:
ha that's fair enough. i'm also tempted to rename the docker compose services (the keys in this yaml file) to match – what do you think?

Reply:
I didn't say that because I didn't want to sound like I was splitting hair but yeah, it's probably wise and less surprising 👍